src/NUnitConsole/nunit3-console.tests/ConsoleRunnerTests.cs

```diff
@@ -11,10 +11,15 @@ namespace NUnit.ConsoleRunner.Tests
 {
     class ConsoleRunnerTests
     {
-        [Test]
+        [Test, Ignore("Needs to be rewritten")]
```

Comment:
We haven't done it before, but we haven't gone back and fixed these either. Maybe we need a general issue for cleaning ignores. I think we have one in the framework. Although, if we do, we still haven't fixed them over a long time.